subgraphs/erc721-holders/schema.graphql

```diff
@@ -109,6 +108,9 @@ type Transfer @entity {
   " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
   logIndex: Int!
 
+  " The subgraph metadata related to this event "
+  subgraph: Subgraph!
```

Comment:
would like some comments on if this is the best way to embed subgraph versioning/metadata in the schema